b2/console_tool.py

```diff
@@ -2402,6 +2405,75 @@ def run(self, args):
         return 0
 
 
+@B2.register_subcommand
+class ReplicationStatus(Command):
+    """
+    Inspects files in only source or both source and destination buckets
+    (potentially from different accounts) and provides detailed replication statistics.
+    """
+
+    @classmethod
+    def _setup_parser(cls, parser):
+        super()._setup_parser(parser)
+        parser.add_argument('source', metavar='SOURCE_BUCKET_NAME')
+        parser.add_argument('--rule', metavar='REPLICATION_RULE_NAME', default=None)
+        parser.add_argument('--destination-profile', default=None)
+        parser.add_argument('--dont-scan-destination', action='store_true')
+        parser.add_argument('--output-format', default='table', choices=('table', 'json'))
+
+    def run(self, args):
+        destination_api = args.destination_profile and _get_b2api_for_profile(
+            args.destination_profile
+        )
+
+        try:
+            bucket = self.api.list_buckets(args.source)[0]
+        except IndexError:
+            self._print_stderr(f'ERROR: bucket "{args.source}" not found')
+            return 1
+
+        if not bucket.replication:
+            self._print_stderr(f'ERROR: no replication set up for bucket "{args.source}"')
+            return 1
+
+        rules = bucket.replication.rules
+        if args.rule:
+            rules = [rule for rule in rules if rule.name == args.rule]
+
+        results_by_rule_name = {}
+        for rule in rules:
+            monitor = ReplicationMonitor(
+                bucket=bucket,
+                rule=rule,
+                destination_api=destination_api,
+            )
+            report = monitor.scan(scan_destination=not bool(args.dont_scan_destination))
```

Comment:
```suggestion
            report = monitor.scan(scan_destination=not args.dont_scan_destination)
```
it's a bool already with implicit `default=False`